js/services/getResource.js

```diff
@@ -1,12 +1,19 @@
-const getResource = async(url) => {
-    const res = await fetch(url, {
-      method: "GET"
-    }) 
-    if (!res.ok) {
-        throw new Error(`Could not fetch ${url}, status ${res.status}`);
-    } else {
-      return await res.json();
-    }
-  };
-  
-  export {getResource};
\ No newline at end of file
+const dbUrl = 'https://irmakdak.github.io/English-for-kids/cards.json';
+function getUrl() {
+  return dbUrl;
+}
+
+const getResource = async () => {
+  const url = getUrl();
+  const res = await fetch(url, {
+    method: 'GET',
+  });
+  if (!res.ok) {
+    throw new Error(`Could not fetch ${url}, status ${res.status}`);
```

Comment:
you throw the error at he place, do you have any handler to catch and manage it?